BuildingThreatModel.md

```diff
@@ -0,0 +1,73 @@
+<!-- SPDX-License-Identifier: CC-BY-4.0 -->
+<!-- Copyright Contributors to the ODPi Egeria project 2020. -->
+# Building a Threat Model
+
+Threat modeling is a process by which potential threats, such as structural vulnerabilities or the absence of appropriate safeguards, can be identified, enumerated, and mitigation can be prioritized. 
+It is recommended that consumers embrace threat modeling technique and walk through series of concrete steps in order to fully understand the environment and specific solution you are trying to build and secure. 
+
+Thread modeling should at the end answer questions like:
+
+- Where am I most vulnerable to attack? 
+- What are the most relevant threats?
+- What do I need to do to safeguard against these threats?
+
+Egeria's architecture allows consumers to make specific choices and minimize risk factors from the early solution design phase.
+Here we outline some generic aspects that can help you to build model specific to your solution using Egeria Open Metadata Platform.
+
+### Decomposing the solution and identifying data flows
+
+- Identify high level application components; 
+- Identify metadata flows;
+    - How metadata is stored?
+    - How metadata is accessed?
+- Identify how Egeria interacts with external entities;
+    - How metadata is exchanged? 
+- Identify Trust boundaries and security measures;
+
+Common data flow patterns between components in Egeria solution:
+
+![Simple data flow diagram](assets/img/Egeria-SimpleDataFlow.png)
+
+### Determine possible threats
+Most common threat categories following [STRIDE](https://docs.microsoft.com/en-us/previous-versions/commerce-server/ee823878(v=cs.20)?redirectedfrom=MSDN) methodology each violating some aspect of the CIA triad.
```

Comment:
I don't know the topic area, but I note from the link that the article is from 2009 and there's a banner 'We're no longer updating this content regularly. Check the Microsoft Product Lifecycle for information about how this product, service, technology, or API is supported.' Not necessarily a problem, but if there's a later good source that would be worth adding

Reply:
Can we also clarify the definition of CIA triad or add a link (I have to admit I had to look it up)